src/main/java/org/jenkinsci/plugins/durabletask/BourneShellScript.java

```diff
@@ -207,6 +188,68 @@ public String getScript() {
         return c;
     }
 
+    private String binaryLauncherCmd(ShellController c, FilePath ws, OsType os, String interpreter, String scriptPath, String cookieValue, String cookieVariable) throws IOException, InterruptedException {
+        String logFile = c.getLogFile(ws).toURI().getRawPath();
+        String resultFile = c.getResultFile(ws).toURI().getRawPath();
```

Comment:
Nope, this will blow up for directories with unusual characters due to URI escaping.

```suggestion
        String resultFile = c.getResultFile(ws).getRemote();
```